tools/mcl/mcl.xml

```diff
@@ -1,5 +1,5 @@
 <tool id="mcl_clustering" name="MCL"  version="14.137">
-    <description>A Markov Cluster Algorithm.</description>
+    <description>A Markov Cluster Algorithm</description>
```

Comment:
What requirement? This PR is about RNAz, isn't it Idon't understand why this change is needed in this PR :)